src/packages/@teams/features/team-explorer/components/workspace-grid-view/WorkspaceGridView.svelte

```diff
@@ -42,239 +37,118 @@
    * Flag to check if user is guest user
    */
   export let isGuestUser = false;
-
-  let workspacePerPage = 5;
+  let filterText = "";
   let currPage = 1;
```

Comment:
don't remove this variable `workspacePerPage`. this will help us to easily set the no of workspace to be shown in a page.